modules/database/database-mongodb-jdbc/src/main/java/org/eclipse/dirigible/mongodb/jdbc/MongoDBConnection.java

```diff
@@ -690,7 +690,7 @@ public SQLXML createSQLXML() throws SQLException {
     public boolean isValid(int timeout) throws SQLException {
         Document response = this.mongoDatabase.runCommand(new BsonDocument("ping", new BsonInt32(1)));
         response.getDouble("ok");
-        return response != null && response.getDouble("ok") == 1.0;
+        return response.getDouble("ok") == 1.0;
```

Comment:
Should we check if `response` is null above? Also, is `response.getDouble("ok")` needed on the line above?

Reply:
fixed

Reply:
It seems that a `Document` object is always returned, so no need from the null check.